dusty/systems/nfs/__init__.py

```diff
@@ -0,0 +1,10 @@
+from . import client
+from . import server
+
+def configure_nfs():
+    server.configure_nfs_server()
+    client.mount_active_repos()
+
+def update_nfs_with_repos(repos):
+    server.add_exports_for_repos(repos)
+    client.remount_repos(repos)
```

Comment:
Is this assuming the server is up and running? We may need to add explicit checks for that and set it up if not. Otherwise the implicit dependency will cause bugs or complication in other parts of the code.